packages/cloudfront-signer/src/sign.spec.ts

```diff
@@ -7,9 +7,18 @@ import { getSignedCookies, getSignedUrl } from "./index";
 const url = "https://d111111abcdef8.cloudfront.net/private-content/private.jpeg";
 const keyPairId = "APKAEIBAERJR2EXAMPLE";
 const dateLessThan = "2020-01-01";
-const epochDateLessThan = Math.round(new Date(dateLessThan).getTime() / 1000);
+const epochDateLessThan = Math.round(
+  (typeof dateLessThan === "string" && /^\d+$/.test(dateLessThan)
+    ? Number(dateLessThan)
+    : new Date(dateLessThan).getTime()) / 1000
+);
```

Comment:
this test code change does not test the runtime code change

Reply:
`dateLessThan` is a constant string. The rounding, the ternary, and the Date constructor don't do anything.

 The new test cases should be:

```js
describe('when signing a URL with a date range')
  it('allows string input compatible with Date constructor')
  it('allows number input compatible with Date constructor')
  it('allows Date input')
```

in each case assert that the getSignedUrl and getSignedCookies outputs are as expected